pkg/kubelet/pod_workers_test.go

```diff
@@ -105,6 +105,11 @@ func (f *fakePodWorkers) ShouldPodRuntimeBeRemoved(uid types.UID) bool {
 	defer f.statusLock.Unlock()
 	return f.removeRuntime[uid]
 }
+func (f *fakePodWorkers) SetPodRuntimeBeRemoved(uid types.UID) {
```

Comment:
The PodWorkers interface does not set `SetPodRuntimeBeRemoved`. 
Does it make sense for this to be a private implementation?

```
f.setPodRuntimeBeRemoved
```

Another question for my own understanding; is what happens in the original implementation? Does this wait for volumes to be unmounted (I understand its in the comments, but wanted to double check).

Reply:
To fix it gracefully, I think we need to use something like `waitForVolumeUnmount` to wait for fake unmount.

`SetPodRuntimeBeRemoved` is more like `unmount runtime by test itself`

Reply:
Updated.